src/components/data_update/DatasetUpdate.jsx

```diff
@@ -153,7 +153,7 @@ export const DatasetUpdate = (props) => {
         validators={[FormValidators.REQUIRED]}
         defaultValue={formData.properties.description}
         onChange={({ value }) => {
-          formData.properties.description = value;
+          formData.properties.duosId = value;
```

Comment:
why are these changing from study columns to dataset columns? Did this work for datasets before because they were nested within a study?

Reply:
Oh, these changes are accidental - good catch